bin/node/runtime/src/lib.rs

```diff
@@ -1937,7 +1938,7 @@ impl_runtime_apis! {
 			address: AccountId,
 			key: [u8; 32],
 		) -> pallet_contracts_primitives::GetStorageResult {
```

Comment:
@athei I guess we should implement `get_storage_transparent()` for the RPC API here?

Reply:
So making `Pallet::get_storage` accept `VarSizedKey` leads to necessity of adding this type to `node-runtime` and `node-rpc` which doesn't seem right way to do to me. After some attempts (see todays commits) I decided not to go in that direction, revert it as it was done at first with `Vec<u8>` and add a new `ContractAccessError` error